dask_cloudprovider/aws/plugins.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+from pathlib import Path
+
+from distributed import WorkerPlugin
+
+logger = logging.getLogger(__name__)
```

Comment:
I have to add some logging for the visibility of the fact that file has been uploaded